pymilvus/client/grpc_handler.py

```diff
@@ -688,7 +688,7 @@ def can_loop(t) -> bool:
             if progress >= 100:
                 return
             time.sleep(DefaultConfigs.WaitTimeDurationWhenLoad)
-        raise MilvusException(message="wait for loading collection timeout")
+        raise MilvusException(message="wait for loading collection timeout, collection " + collection_name)
```

Comment:
can be simplified to:
`f"wait for loading collection timeout, collection: {collection_name}"`